health/micro-ui/web/micro-ui-internals/packages/modules/campaign-manager/src/pages/employee/MyMicroplans.js

```
@@ -0,0 +1,32 @@
+    <React.Fragment>
+
+      <Header styles={{ fontSize: "32px" }}>{t("MY_MICROPLANS_HEADING")}</Header>
+      <div className="inbox-search-wrapper">
+        <InboxSearchComposer
+        configs={MicroplanCampaignSearchConfig?.[0]}
+        additionalConfig={{
+          resultsTable: {
+            onClickRow,
+          },
+        }}
+        ></InboxSearchComposer>
```

Biome

[error] JSX elements without children should be marked as self-closing. In JSX, it is valid for any element to be self-closing.

Unsafe fix: Use a SelfClosingElement instead


(lint/style/useSelfClosingElements)